backend/resources/participant.py

```diff
@@ -29,6 +29,21 @@ def post(self):
         except ValidationError as err:
             return (err.messages), HTTPStatus.BAD_REQUEST
 
+        if Participant.query.filter(Participant.email == json_data["email"]).first():
+            return (
+                {"message": "User with this email already exists."},
+                HTTPStatus.CONFLICT,
+            )
+
+        if (
+            Participant.query.filter(Participant.github == json_data["github"]).first()
+            is not None
+        ):
```

Comment:
I think that `is not None` could be removed.
Also You could use here:
`if Participant.query.filter_by(github=json_data["github"]).first():`